src/api/hooks/useGetSearchResults.ts

```diff
@@ -54,7 +57,7 @@ export default function useGetSearchResults(input: string) {
               label: `Account ${truncateAddress(address)}${
                 primaryName ? ` | ${primaryName}.apt` : ``
               }`,
-              to: `/account/${address}`,
+              to: augmentToWithGlobalSearchParams(`/account/${address}`),
```

Comment:
couple other places needs to use `augmentToWithGlobalSearchParams` as well, i.e. https://github.com/aptos-labs/explorer/blob/91bd8d92df6bfd4e2261c6a5f17196555c4ca242/src/api/hooks/useGetSearchResults.ts#L90.

Reply:
actually I'm already augmenting the url for _all_ results on line 152. That being said the augmentation on line 60 is kinda unnecessary since it's being double augmented, so I'm just removing it from line 60 to avoid confusion.